clickhouse-client/src/test/java/com/clickhouse/client/BaseIntegrationTest.java

```diff
@@ -5,19 +5,20 @@
 import java.util.Locale;
 import java.util.Map;
 
+import com.clickhouse.client.config.ClickHouseClientOption;
 import org.testng.annotations.BeforeTest;
 
 /**
  * Base class for all integration tests.
  */
 public abstract class BaseIntegrationTest {
-    @BeforeTest(groups = { "integration" })
+    @BeforeTest(groups = {"integration"})
     public static void setupClickHouseContainer() {
         ClickHouseServerForTest.beforeSuite();
     }
 
     protected ClickHouseNode getSecureServer(ClickHouseProtocol protocol) {
-        return ClickHouseServerForTest.getClickHouseNode(protocol, true, ClickHouseNode.builder().build());
+        return ClickHouseServerForTest.getClickHouseNode(protocol, true, ClickHouseNode.builder().addOption(ClickHouseClientOption.SSL.getKey(), "true").build());
```

Comment:
Is the ssl option still needed? You may remove this one or duplicates in other tests.